chaviprom/secure_otp_utils.py

```
@@ -0,0 +1,216 @@
+django-two-factor-auth against OTP bypass attacks. These functions
+work with the existing middleware to provide enhanced security.
+"""
+
+import hashlib
+import hmac
+import time
+import logging
+import os
+from django.conf import settings
+from django.core.cache import cache
+from django.http import JsonResponse
```

Ruff (0.12.2)

`django.http.JsonResponse` imported but unused

Remove unused import: `django.http.JsonResponse`

(F401)